src/agent/communicator/src/http_client_utils.hpp

```diff
@@ -1,71 +1,112 @@
+#pragma once
 #include <ihttp_socket.hpp>
 
 #include <boost/asio.hpp>
 #include <boost/beast.hpp>
 #include <boost/system.hpp>
+#include <logger.hpp>
 
-#include <fstream>
-#include <ios>
+#include <memory>
 #include <string>
 
 namespace http_client_utils
 {
-    /// @brief Reads a response from a socket and writes it to a file
-    /// @param socket The socket to read from
-    /// @param res The response to use
-    /// @param dstFilePath The path to the file to write to
+    /// @brief Timeout for HTTP requests
+    constexpr int TIMEOUT_SECONDS = 30;
```

Comment:
To avoid a race condition with https://github.com/wazuh/wazuh/blob/69b9d9f7829e5af297fe24db5c94a3b0dc4b64a8/apis/comms_api/routers/commands.py#L13, let's increase this value to 60.
```suggestion
    constexpr int TIMEOUT_SECONDS = 60;
```

Reply:
Done